pkg/runner/flags.go

```diff
@@ -130,3 +140,33 @@ func ParseFlags(cfg *Config, args []string) (*Config, error) {
 	cfg.PgmodelCfg.UsesHA = cfg.HaGroupLockID != 0
 	return cfg, nil
 }
+
+func validate(cfg *Config) error {
+	if err := api.Validate(&cfg.APICfg); err != nil {
+		return fmt.Errorf("error validating API configuration: %w", err)
+	}
+	if err := validateMultiTenancyFlags(cfg); err != nil {
+		return fmt.Errorf("error validating mlti-tenancy configuration: %w", err)
+	}
+	return nil
+}
+
+func validateMultiTenancyFlags(cfg *Config) error {
+	if cfg.ValidTenantsListStr != "" {
+		cfg.ValidTenantsList = strings.Split(cfg.ValidTenantsListStr, ",")
+		cfg.ValidTenantsList = removeEmptyTenants(cfg.ValidTenantsList)
```

Comment:
I think cfg.ValidTenantList should already be a type that is explicit between "allow all" and empty list

Reply:
here is the purpose is to catch `tenant-a, tenant-b, ` (note the trailing `,`)